vendors/cypress/boards/CY8CKIT_064S0S2_4343W/aws_demos/config_files/mqtt_demo_connection_sharing_config.h

```diff
@@ -92,6 +92,13 @@
  * #define democonfigMQTT_BROKER_PORT    ( insert here. )
  */
 
+/**
+ * @brief The number of simple subscribe-publish tasks to create for the demo
+ */
+#define democonfigNUM_SIMPLE_SUB_PUB_TASKS_TO_CREATE       1
+#define democonfigSIMPLE_SUB_PUB_TASK_STACK_SIZE           ( configMINIMAL_STACK_SIZE )
+
```

Comment:
Should there also be greyed out config documentation for `MQTT_AGENT_NETWORK_BUFFER_SIZE` and `MQTT_AGENT_COMMAND_QUEUE_LENGTH` macros as is the case for the connection setting macros?

Reply:
Sure